README.md

```diff
@@ -154,6 +154,42 @@ flowchart LR
   end
 ```
 
+## Container images
+
+The project includes container images for running the pipelines. Particularly, the GitHub Action workflow located in [.github/workflows/containers-and-az-pool.yaml](.github/workflows/containers-and-az-pool.yaml) automatically builds an image based on [Dockerfile](Dockerfile) and pushes it to Azure Container Registry. The images can also be built locally, in which case the [Makefile](Makefile) included in the project contains the following targets:
```

Comment:
Out of curiosity, could we have the CI workflow just use the make target to simplify total code? Or are there too many differences between CI and local?

Reply:
Yes I would like this if possible but not sure we want to replace the built-in build and push action.

Reply:
When you say built-in build and push, you mean the pre-built docker CI commands?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/blob/a320d885c09e59d06e88c175dd42ed010e12f544/.github/workflows/containers-and-az-pool.yaml#L83

Reply:
yea, that looks a little painful to replace at the moment